locations/spiders/poczta_polska_pl.py

```diff
@@ -50,6 +50,8 @@ def parse2(self, response):
         ).replace(" ", "")
         addr = response.xpath("//div[contains(@class, 'pp-map-tooltip__adress')]//p/text()").getall()
         item["street"], item["housenumber"] = addr[0].rsplit(" ", 1)
+        if isinstance(item["housenumber"], str) and item["housenumber"].lower() == "b/n":
```

Comment:
I can't tell off the top of my head if house number will always be a string, or if it might ever be None given its the result of a split of another string